hivemind.yaml

```
@@ -0,0 +1,37 @@
+package:
+  name: hivemind
+  version: 1.1.0
+  epoch: 0
+  description: "Process manager for Procfile-based applications"
+  copyright:
+    - license: MIT
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - gobump
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/DarthSim/hivemind
+      tag: "v${{package.version}}"
+      expected-commit: 580abe5b3faf585c450604227e40e960cdbb21bd
+
+  - runs: gobump --packages golang.org/x/sys@v0.0.0-20220412211240-33da011f77ad
```

Can we switch to using the built in go bump pipeline?  Once hivemind updates it's own dependency the the Wolfi automation will remove the go bump section. 

```suggestion
      - uses: go/bump
        with:
          deps: golang.org/x/sys@v0.0.0-20220412211240-33da011f77ad
```